tests/customer_factory.py

```diff
@@ -16,9 +16,23 @@
 Test Factory to make fake objects for testing
 """
 import factory
-from factory.fuzzy import FuzzyChoice
+from factory.fuzzy import FuzzyChoice, FuzzyInteger
 from service.models import Customer
 
 class CustomerFactory(factory.Factory):
     """ Creates fake customers that you don't have to feed """
-    
+    class Meta:
+        model = Customer
+    id = factory.Sequence(lambda n: n)
+    first_name = factory.Faker('first_name')
+    last_name = factory.Faker('last_name')
+    address = factory.Faker('address')
+    customer_id = FuzzyInteger(1,100) # Assume that the customer_id is from 1 to 100
+    user_id = FuzzyText(8) # Assume that the length of user_id is 8
+    password = FuzzyText(8) # Assume that the length of password is 8
+    available = FuzzyChoice(choices=[True, False])
```

Comment:
In the model, this parameter is called active.

Reply:
You mean "available"?

Reply:
Yes. We have active instead of available